docs/usage/session.rst

```diff
@@ -0,0 +1,71 @@
+.. _session:
+
+==================
+Session management
+==================
+
+Zyte API provides powerful session APIs:
+
+-   :ref:`Client-managed sessions <zyte-api-session-id>` give you full control
+    over session management.
+
+-   :ref:`Server-managed sessions <zyte-api-session-contexts>` let Zyte API
+    handle session management for you.
+
+When using scrapy-zyte-api, you can use these session APIs through the
+corresponding Zyte API fields (:http:`request:session`,
+:http:`request:sessionContext`).
+
+However, scrapy-zyte-api also provides :ref:`its own session API
+<plugin-sessions>`.
+
+.. _plugin-sessions:
+
+scrapy-zyte-api’s session API
+=============================
+
+scrapy-zyte-api’s session API offers a higher-level API on top of
+:ref:`client-managed sessions <zyte-api-session-id>`, to enjoy some of the
+advantages of client-managed sessions (e.g. expiring specific sessions) while
+removing some of their drawbacks (management overhead).
+
+To use scrapy-zyte-api’s session API, define
+:setting:`ZYTE_API_SESSION_PARAMS` or :setting:`ZYTE_API_SESSION_CHECKER`.
+
+Often it makes sense to define both. For example:
+
+.. code-block:: python
+    :caption: settings.py
+
+    from scrapy import Request
+    from scrapy.http.response import Response
+
+    ZYTE_API_SESSION_PARAMS = {
+        "browserHtml": True,
+        "actions": [
+            {
+                "action": "setLocation",
+                "address": {"postalCode": "04662"},
+            }
+        ],
+    }
+
+
+    class MySessionChecker:
+
+        @classmethod
+        def from_crawler(cls, crawler):
+            return cls(crawler)
+
+        def __init__(self, crawler):
+            params = crawler.settings["ZYTE_API_SESSION_PARAMS"]
```

Comment:
Wouldn't it make sense to be able to set a configuration per netloc? So, in case `setLocation`does not work in the website you may be able to pass the exact action sequence.

Reply:
> Wouldn't it make sense to be able to set a configuration per netloc?

Yes, but it is my understanding that, for this initial version, we are OK with a solution that works on spiders that target a specific netloc (and even a specific ZIP code). Would you prefer to address this already?

Reply:
As long as we can extend it later to support it it's fine.

Reply:
I'd rather have a way to set a zip code from the very beginning.